lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -47,6 +47,8 @@ class ReportRenderer {
     // Mutate the UIStrings if necessary (while saving originals)
     const originalUIStrings = JSON.parse(JSON.stringify(Util.UIStrings));
 
+    this._dom.setLighthouseChannel(result.configSettings.channel);
```

Comment:
can you add a test to `report-renderer-test.js` as well that if `channel` is in the LHR it ends up on the links?

Reply:
> if channel is in the LHR it ends up on the links?

something like

```js
it('should add LHR channel to doc link parameters', () => {
  const lhrChannel = sampleResults.configSettings.channel;
  // Make sure we have a channel in the LHR.
  assert.ok(lhrChannel.length > 2);

  const container = renderer._dom._document.body;
  const output = renderer.renderReport(sampleResults, container);

  const utmChannels = [...output.querySelectorAll('a[href*="utm_source=lighthouse"')]
    .map(a => new URL(a.href))
    // Filter out known origins we don't care about.
    .filter(url => url.origin !== 'https://snyk.io')
    .map(url => url.searchParams.get('utm_medium'));

  assert.ok(utmChannels.length > 20);
  utmChannels.forEach(anchorChannel => {
    assert.strictEqual(anchorChannel, lhrChannel);
  });
});
```
which should survive us adding more supported origins (e.g. web.dev) or changing the `Learn more` text but forgetting to update this test